e2e/test/scenarios/models/reproductions/29951-model-editor-results-metadata.cy.spec.js

```diff
@@ -55,8 +61,13 @@ const removeExpression = name => {
 
 const dragColumn = (index, distance) => {
   cy.get(".react-draggable")
+    .should("have.length", 20) // 10 columns X 2 draggable elements
```

Comment:
Without this, the UI shows some intermediary "ghost" column headers.
Look at the top of the screenshot.

![image](https://github.com/metabase/metabase/assets/31325167/b33bda4d-d50e-4130-8ab7-b81bba8c2687)